### PR TITLE
feat: add templates

### PR DESCRIPTION
**Pull request by cstrnt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adds new Issue Templates for this repo

**What is the current behavior? (You can also link to an open issue here)**
There are no templates in place

**What is the new behavior (if this is a feature change)?**
There are different templates for different occasions

**Additional info:**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/cli_request.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: CLI Request
+about: Is there a CLI which doesn't have autocompletion? Request it here!
+labels: "CLI Request"
+---
+
+### CLI Tool
+
+What CLI tool is missing autocompletion in fig?
```
Comment (mattschrage) [inline .github/ISSUE_TEMPLATE/cli_request.md:9]: Can you capitalize Fig here?

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Enhancement
+about: You have an Idea on how to improve fig? Let us know!
```
Comment (mattschrage) [inline .github/ISSUE_TEMPLATE/enhancement.md:3]: Update capitalization!

File: .github/ISSUE_TEMPLATE/other.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+### Question
+
+Do you have a Question? Make sure to check out our forum [here](https://forum.withfig.com/)
+
+
+### Community
+
+Do you want to get updates or chat with the creators and maintainers from fig? Check out our community slack [here](https://figcommunity.slack.com/join/shared_invite/zt-fupa9n8g-sfHm8MyBn1DBaCj8SoIxSA#/shared-invite/email)
```
Comment (mattschrage) [inline .github/ISSUE_TEMPLATE/other.md:14]: Update capitalization of Fig and Slack.

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+labels: "Enhancement"
+---
+
+### Completion Spec
+
+What completion spec does this enhancement refer to? Or is it fig itself?
```
Comment (mattschrage) [inline .github/ISSUE_TEMPLATE/enhancement.md:9]: Capitalization!

File: .github/ISSUE_TEMPLATE/bug.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+  Useful Links:
+  - Documentation: https://withfig.com/docs/autocomplete/getting-started
+
+  Before opening a new issue, please search existing issues: https://github.com/withfig/autocomplete/issues
+-->
+
+### Completion Spec
+
+What completion spec does this bug refer to? Or is it fig itself?
```
Comment (mattschrage) [inline .github/ISSUE_TEMPLATE/bug.md:18]: Capitalize Fig!

**Review by mattschrage — CHANGES_REQUESTED** [review]
Looks good! Just fix up the capitalization issues.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report. For questions, please use "Discussions".
```
Comment (QuiiBz) [inline .github/ISSUE_TEMPLATE/bug.md:3]: The repo doesn’t have Discussions enabled yet. Is it planned to be added?

File: .github/ISSUE_TEMPLATE/cli_request.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+labels: "CLI Request"
+---
+
+### CLI Tool
+
+What CLI tool is missing autocompletion in Fig?
+
+### Useful Links
+
+Please add some links to the docs of that tool or the `--help`text
```
Comment (QuiiBz) [inline .github/ISSUE_TEMPLATE/cli_request.md:13]: Missing final dot.

**Pull request merged**